packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -22,6 +22,160 @@ async function instantiatePrism(specPath: string) {
   return server;
 }
 
+// Note that these test cases cover the situation describe in https://stoplightio.atlassian.net/browse/SO-265
```

Comment:
Contributors cannot see what this is so we're confusing them with a TODO not they cant read. If something is worth sharing can we copy and paste it somewhere accessible?